pkgs/games/eduke32/default.nix

```diff
@@ -58,7 +58,7 @@ in stdenv.mkDerivation rec {
     done
   '';
 
-  NIX_CFLAGS_COMPILE = "-I${SDL2.dev}/include/SDL2 -I${SDL2_mixer}/include/SDL2";
+  NIX_CFLAGS_COMPILE = "-I${SDL2.dev}/include/SDL2 -I${SDL2_mixer.dev}/include/SDL2";
```

Comment:
```suggestion
  NIX_CFLAGS_COMPILE = "-I${SDL2.dev}/include/SDL2 -I${lib.getDev SDL2_mixer}/include/SDL2";
```
and for the others to make output changes in SDL2_mixer not require downstream changes

Reply:
That's a good question. `lib.get*` are probably best to use when you are not sure of interface stability (like some libcs provide `.dev`, some don't). This example is arguably similar if we have to ever roll it back (if some packages can't survive `lib`/`out` split across different prefix). For some more established cases it's probably ok to use output directly.

I'll convert to `lib.getDev`.

Reply:
Switched to `lib.getDev`.